src/app/pages/system/general-settings/general-settings.component.ts

```diff
@@ -267,52 +268,54 @@ export class GeneralSettingsComponent implements OnInit {
 
   saveConfigSubmit(entityDialog: EntityDialogComponent): void {
     entityDialog.loader.open();
-    entityDialog.ws.call('system.info').pipe(untilDestroyed(this)).subscribe((systemInfo) => {
-      let fileName = '';
-      let mimetype: string;
-      if (systemInfo) {
-        const hostname = systemInfo.hostname.split('.')[0];
-        const date = entityDialog.datePipe.transform(new Date(), 'yyyyMMddHHmmss');
-        fileName = hostname + '-' + systemInfo.version + '-' + date;
-        if (entityDialog.formValue['secretseed']) {
-          mimetype = 'application/x-tar';
-          fileName += '.tar';
-        } else {
-          mimetype = 'application/x-sqlite3';
-          fileName += '.db';
+    this.store$.pipe(waitForSystemInfo, untilDestroyed(this)).subscribe(
+      (systemInfo) => {
+        let fileName = '';
+        let mimetype: string;
+        if (systemInfo) {
```

Comment:
seems like this check is unnecessary

Reply:
> Like this idea. The changes look good except for one issue:
> Kapture.2022-05-16.at.14.19.09.mp4
> 
> For some reason, the dashboard page is still empty and no errors in logs were tested against a cloud VM.

The issue with the empty dashboard is only an issue of slow network/slow cloud vm responses. If you wait long enough, the cloud vm responds and the dashboard shows up. Maybe we should have a loading indication on the dashboard too.

Reply:
Yeah, it might be. It just happened in this branch. When opened cloud VM on the same network directly, it shows correctly. I'll check on a local VM and let you know 👍🏻 
The loading indicator will be addressed in #6661

Reply:
Sounds good. Please do.